browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -3540,17 +3540,15 @@ index 7a790bb23c8f37af57e4284b367cad9d679a434f..e1c81580f7756c722e0ba6caaff23073
  {
      Document* document = &node->document();
      if (auto* templateHost = document->templateDocumentHost())
-@@ -2848,12 +2982,17 @@ RefPtr<Protocol::Runtime::RemoteObject> InspectorDOMAgent::resolveNode(Node* nod
-     if (!frame)
+@@ -2849,11 +2983,16 @@ RefPtr<Protocol::Runtime::RemoteObject> InspectorDOMAgent::resolveNode(Node* nod
          return nullptr;
  
--    auto& globalObject = mainWorldGlobalObject(*frame);
+     auto& globalObject = mainWorldGlobalObject(*frame);
```

Comment:
Let's use globalObject at line 3558 below: `return injectedScript.wrapObject(nodeAsScriptValue(globalObject, node), objectGroup);` it should eliminate that part of the diff.